### DISCUSSION TITLE
Add a Dutch provider

### DISCUSSION THREAD

**Discussion by fraternl** [Freetz-NG/freetz-ng#898] [Allgemein]
On my private connection I have  a 7530 given by the fibre provider which has its own profile in the selectable list.

This provider is not on standard modern Fritzboxes. 
I'm thinking of replacing it with a Freetzed 7530AX in the near future.

I already did an experiment over a weekend with a Freetzed 7590. I had to call the provider to find out which vlan I would need.
They have a vlan 500 & 501 respectively for Internet and IPTV.

The provider is called VAIBR and isn't big, but still available in several places. The whole hometown has FTTH, not all of them have a subscription. I like them because they don't force you to use setup-boxes for TV. The fibre-modem has 1 Gbit UTP-ethernet and a coax with DVB-C and and analog TV.

I've created this profile based on another one.
I don't think it contains errors, but it isn't tested yet.

It's based on the export confiruration of the non-Freetzed 7530 export file.
I'm only interested in it if it then becomes part of Freetz-ng.
For my own Freetz I can just do a manual configuration.

```
ar7cfg {
        mode = dsldmode_router;
        vccs {
                dsl_encap = dslencap_mixed;
                connections = "internet", "iptv";
        }
        mcupstream = "iptv";
        dslifaces {
                enabled = yes;
                name = "internet";
                dsl_encap = dslencap_ether;
                vlancfg {
                        vlanencap = vlanencap_fixed_prio;
                        tagtype = vlantagtype_customer;
                        vlanid = 500;
                        vlanprio = 1;
                }
                ppptarget = "internet";
                is_mcupstream = no;
                stay_always_online = yes;
        } {
                enabled = no;
                name = "iptv";
                dsl_encap = dslencap_ether;
                vlancfg {
                        vlanencap = vlanencap_fixed_prio;
                        tagtype = vlantagtype_customer;
                        vlanid = 501;
                        vlanprio = 5;
						tos = 0;
                }
                ppptarget = "iptv";
                is_mcupstream = yes;
                stay_always_online = yes;
                }
                dsldpconfig {
                        security = dpsec_firewall;
                        filter_teredo = yes;
                        filter_netbios = yes;
                        filter_sip = no;
                        filter_smtp = no;
                        sip_alg = no;
                        lowinput {
                                policy = "permit";
                        }
                        lowoutput {
                                policy = "permit";
                        }
                        highinput {
                                policy = "permit";
                        }
                        highoutput {
                                policy = "permit";
                        }
                }
        }
        targets {
                type = pppcfg_target_internet;
                name = "internet";
        } {
                type = pppcfg_target_internet;
                name = "iptv";
        }
        dslglobalconfig {
                autodetect = no;
                Annex = "A";
                pppoeiface = "eth0";
                speed_in_netto = 1000000;
                speed_out_netto = 1000000;
        }
}

```

**Comment by fraternl** [comment]
BTW...
I think I can recover the provider list of the non-Freetzed 7530 with "recover"  can I?

**Comment by fda77** [comment]
A recovery reverts everything, but to restore a settings backup should be enough for ar7.cfg changes

**Comment by fraternl** [comment]
Yes, I know the provider files are the ar7.cfg and tr086.cfg, but they're only an excerpt.

If you load an export to a Fritzbox which has a predefined profile that's not in the image, it will end up with an unconfigured WAN.

What I posted is probably what's in the ar7.cfg that they have merged into their Fritzboxes they distribute to their customers.

I took an ar7.cfg of another provider and edited it.

My plan is to load the configuration of the 7530 into a Freetzed 7530AX and then manually configure it with vlan 500 (that's because the factory 7530AX has no predefined VAIBR profile... been there).

I then take the 7530 to work and recover the image and extract the provider files out of the image.

I then incorporate the VAIBR files into my own image and flash a random Fritzbox and check if it accepts the config of the 7530 and still have the WAN configured. The latter is not the case on normal images.

How VAIBR can update official AVM updates is yet a mystery to me. Maybe AVM provides a way to slipstream them. I would think it would be easier for that provider to ask AVM to include them

**Comment by fda77** [comment]
Some device have  so called "provider additive" which could help you. Search IPPF for more informations
